### ISSUE TITLE
[HW3] 컴포넌트 종류 및 hooks 관련 질문

### ISSUE THREAD

**Issue by curious-curie** [swsnu/swppfall2020#74]
안녕하세요.
hw3 관련 질문 드립니다.
redux 의 전역 state와 연결하거나 라이프사이클 메서드 사용하는 컴포넌트는 실습 때 배운 방식 그대로 클래스형 컴포넌트를 사용해야 하는지 여쭤보고 싶습니다.
컴포넌트 구현 시 실습시간에 배운 클래스형 컴포넌트 뿐 아니라 함수형 컴포넌트로 구현하고 mapStateToProps, connect 대신 hooks 의 useDispatch, useSelector을 사용하는 방식으로 구현을 해도 되나요?

감사드립니다! ☺️

**Comment by hy00nc** [comment]
네, 말씀하신 대로 react, redux을 사용하면서 스펙만 잘 지켜진다면 구현 방식은 자유롭게 선택하셔도 됩니다.
